_troubleshooting/delete_security_group.md

```diff
@@ -10,7 +10,7 @@ dateAdded: June 24, 2016
 
 ### Q. Why can't I delete a security group?
```

Comment:
May be worth showing an example of the error message some one would receive if they attempt to delete a security group but can't. That way if some one searches for that error in the docs, they'll end up at this page.